services/search/README.md

```diff
@@ -52,6 +52,8 @@ As soon as Tika is installed and accessible, the search service must be configur
 
 When the search service can reach Tika, it begins to read out the content on demand. Note that files must be downloaded during the process, which can lead to delays with larger documents.
 
+Content extraction and handling the extracted content can be very resource intensive. Therefor content extraction is limited to files with a certain file size. The default limit is 20MB and can be configured using the `SEARCH_CONTENT_EXTRACTION_SIZE_LIMIT` variable.
```

Comment:
```suggestion
Content extraction and handling the extracted content can be very resource intensive. Content extraction is therefore limited to files with a certain file size. The default limit is 20MB and can be configured using the `SEARCH_CONTENT_EXTRACTION_SIZE_LIMIT` variable.
```
Therefore is an adverb that means “as a consequence,” “as a result,” or “hence.”
Therefor is an adverb that means “for that,” or “for it.”